R/quantity-type.R

```diff
@@ -14,3 +14,12 @@ QuantityType <- enum(list(
   Parameter = bitwShiftL(2, 8),
   Time = bitwShiftL(2, 9)
 ))
+
+getQuantityTypesStringFromInt <- function(quantityInt){
+  quantityTypes <- intToBits(quantityInt)
+  quantityTypes <- which(as.logical(quantityTypes))
+  quantityTypes <- bitwShiftL(2, quantityTypes-2)
+  quantityTypes <- lapply(quantityTypes, getEnumKey, QuantityType)
+
+  return(quantityTypes)

```

Comment:
I would rename this function to getQuantityTypeAsString and return the string instead of the list. Comsumer will always have to paste and collapse which is not ergnomic

Reply:
Done.